Library/Model/SPITerminal.h

```diff
@@ -0,0 +1,77 @@
+//
+//  SPITerminalStatus.h
+//  Pods
+//
+//  Created by Metin Avci on 24/9/18.
+//
+
+#import <Foundation/Foundation.h>
+#import "SPIMessage.h"
+
+@interface SPITerminalBattery : NSObject
+
+@property (nonatomic, readonly, strong) NSString *batteryLevel;
```

Comment:
Just FYI, if you've specified `readonly`, all these other specifiers like nonatomic/strong don't matter any more because they only apply to the synthesized setter (which doesn't exist if its readonly). No big deal, no need to fix, just a learning opportunity :)
```suggestion
@property (readonly) NSString *batteryLevel;
```

Reply:
Thank you for suggestion